avahi-daemon/chroot.c

```diff
@@ -188,8 +188,8 @@ static int recv_fd(int fd) {
             return -1;
         }
 
-        assert(h->cmsg_len = CMSG_LEN(sizeof(int)));
-        assert(h->cmsg_level = SOL_SOCKET);
+        assert(h->cmsg_len == CMSG_LEN(sizeof(int)));
+        assert(h->cmsg_level == SOL_SOCKET);
```

Comment:
This part has been already proposed in other PR. Have to find its ID.

Reply:
This is also fixed in PR #372 .

Reply:
So let's close #372 as a duplicate of this one?